routes/movies.js

```diff
@@ -0,0 +1,97 @@
+const Celebrity = require("../models/Celebrity.model");
+const Movie = require("../models/Movie.model");
+const router = require("express").Router();
+
+router.get("/", (req, res) => {
+  Movie.find()
+    .then((allMovies) => {
+      res.render("../views/movies/movies", { allMovies });
```

Comment:
I would go with that syntax so it's a little shorter

```suggestion
      res.render("movies/movies", { allMovies });
```